### PR TITLE
KSP physics

### PR DESCRIPTION
**Pull request by eggrobin**
A separate DLL with instantiations of `Ephemeris`, `ContinuousTrajectory`, and `Body` and its subclasses, for optimization and benchmarking.

This requires mockingbirdnest/protobuf#3.

### CODE REVIEW

File: Principia.sln [hunk 1]
```diff
@@ -467,6 +469,27 @@ Global
 		{A0E67E1B-E5A6-45A0-B42C-4330A6643CD7}.Release|Win32.Build.0 = Release|Any CPU
 		{A0E67E1B-E5A6-45A0-B42C-4330A6643CD7}.Release|x64.ActiveCfg = Release|Any CPU
 		{A0E67E1B-E5A6-45A0-B42C-4330A6643CD7}.Release|x64.Build.0 = Release|Any CPU
+		{3882467C-89D7-4DAE-9730-BB2E610F05AD}.Debug|Any CPU.ActiveCfg = Debug|Win32
```
Comment (pleroy) [inline Principia.sln]: Remove all the `Any CPU`.

File: ksp_physics/ksp_physics_lib.cpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                             &module_info,
+                             sizeof(module_info)));
+  LOG(INFO) << "Base address is " << module_info.lpBaseOfDll;
+
+  google::protobuf::SetLogHandler([](google::protobuf::LogLevel const level,
+                                     char const* const filename,
+                                     int const line,
+                                     std::string const& message) {
+    LOG_AT_LEVEL(level) << "[" << filename << ":" << line << "] " << message;
+  });
```
Comment (pleroy) [inline ksp_physics/ksp_physics_lib.cpp]: Remove.

Comment (eggrobin) [reply ksp_physics/ksp_physics_lib.cpp]: Done.

File: Principia.sln (hunk 1 above)
Comment (eggrobin) [reply Principia.sln]: Done.

**Pull request merged**